src/smartcontracts/code.ts

```diff
@@ -25,4 +37,12 @@ export class Code {
     valueOf(): Buffer {
         return Buffer.from(this.hex, "hex");
     }
+
+    computeHash(): Hash {
```

Comment:
I think this can directly be a `Buffer`. At some point, we might remove the `Hash` object. E.g. see the sdk-specs for transaction hash: https://github.com/multiversx/mx-sdk-specs/blob/main/sdk-core/transaction.md

Reply:
Updated